src/vue/backend/src/templates/importHeader.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <idx-tab-container
+        customClass="import-header"
+        @activeTab="switchTabs"
+        :activeTab="activeTab"
+        :tabs="['UNIMPORTED', 'IMPORTED']"
```

Comment:
All caps should be a CSS thing only unless declaring constants. Remember that screen readers consider them acronyms.